app/src/main/res/values/strings.xml

```diff
@@ -401,9 +401,9 @@
     <string name="pref_import_existing_msg">Add components disabled by others apps to App Manager.
         Be careful when using this tool as there can be many false positives. Choose only the apps you\'re certain about.</string>
     <string name="clear_cache">Clear Cache</string>
-    <string name="block_unblock_trackers_description">Block or unblock ad and tracking components from all the installed apps</string>
+    <string name="block_unblock_trackers_description">Allow or deny ad and tracking components in all the installed apps</string>
```

Comment:
True, but the title already says block, since allow or deny is also true in the context of what blocking does, it helps the user understand beyond just "block"

Reply:
Nope. Allow/deny is completely different. Components can be blocked/unblocked (via IFW) or enabled/disabled (via PackageManager). They can't be allowed or denied like app ops. As I've stated in a previous PR, I like to use Android terminologies rather than creating new ones as the users of the app are expected to be familiar with Android to some extent or have already read the docs.